src/Policies/Policy.php

```diff
@@ -55,6 +55,24 @@ public function applyTo(Response $response): void
         }
 
         $response->headers->set($headerName, (string) $this);
+
+        if (! config('feature-policy.reporting.enabled')) {
+            return;
+        }
+
+        $response->headers->set('Reporting-Endpoints', 'violation-reports="' . config('feature-policy.reporting.url') . '"');
+
+        if (! config('feature-policy.reporting.report_only')) {
+            return;
+        }
+
+        $headerName = 'Permissions-Policy-Report-Only';
```

Comment:
this is a duplicate of the regular header, but instructs the browser to only inform the user and to not send a report.